src/server/services/wms/qgswmsutils.cpp

```diff
@@ -157,7 +157,7 @@ namespace QgsWms
         break;
       default:
         QgsMessageLog::logMessage( QString( "Unsupported format string %1" ).arg( formatStr ) );
-        saveFormat = UNKN;
+        saveFormat = "Unknown";
```

Comment:
Is `UNKN` some define/variable that is still used somewhere or something that we can remove?

Reply:
I think the "default:" here should actually be catching "UNKN" instead, but I'll let one of the server team confirm that...

Reply:
Yep, it was supposed to be catching UNKN here. I've fixed that now.